src/Trie.mo

```diff
@@ -129,44 +131,26 @@ module {
     }
   };
 
-  /// Checks the invariants of the trie structure, including the placement of keys at trie paths
-  public func isValid<K, V>(t : Trie<K, V>, enforceNormal : Bool) : Bool {
+  public func assertIsValid<K, V>(t : Trie<K, V>) {
+    assert isValid(t)
+  };
+
+  public func isValid<K, V>(t : Trie<K, V>) : Bool {
```

Comment:
Sorry, Matthew, but this is still a breaking change since you remove the enforceNormal parameter.

I'd add that back in and remove the new `assertIsValid` which is pretty redundant. Let's not add random functions with little added benefit. A user can can just do the assert themselves.

I found the breakage by actually trying the code against the repro on the playground. After adjusting the calls, the repro seems to work just fine.